js/MineSweeper.js

```diff
@@ -79,86 +82,127 @@ jQuery(function ($) {
             }
             // insert progress animation before the grid
             if ($('.ajax-loading').length < 1) {
-                $('<div class="invisible ajax-loading"></div>')
-                    .prependTo(self.element.parent());
-            }
-            self.initWorkers();
-            self.clearBoard();
-            self.redrawBoard();
-            self.resetDisplays();
-            self.initHandlers();
-            return self;
+				msUI.before(
+                '<div class="invisible ajax-loading"></div>'
+				);
+            }
+            msObj.initWorkers(msObj.options.path_to_cell_toucher);
+            msObj.clearBoard();
+            msObj.redrawBoard();
+            msObj.resetDisplays();
+            msObj.initHandlers(msUI);
+            return msObj;
         };
 
-        this.initWorkers = function () {
+		this.callWorker = function(task,par) {
+            $('.ajax-loading').removeClass('invisible');
+            var job = {
+                type: task, // message type
+                grid: msObj.grid
+            };
+			if (typeof par === 'number') {
```

Comment:
Can you fix the indenting on these lines? It looks misaligned. I think the standard should be 4 spaces per indent